### PR TITLE
modules/SceNet: implement LLE libHttp and libSsl

### PR DESCRIPTION
**Pull request by bookmist**
I implement functions, needed for LLE libHttp and LibSsl.
Its ready for review and merge

### CODE REVIEW

**Comment by Zangetsu38** [comment]
Internal request another librairy file

File: vita3k/modules/SceDriverUser/SceAppMgrUser.cpp [hunk 1]
```diff
@@ -140,9 +140,11 @@ EXPORT(int, sceAppMgrContentInstallPeriodStop) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppMgrConvertVs0UserDrivePath) {
-    TRACY_FUNC(sceAppMgrConvertVs0UserDrivePath);
-    return UNIMPLEMENTED();
+EXPORT(int, sceAppMgrConvertVs0UserDrivePath, char *source_path, char *dest_path, int dest_len) {
+    TRACY_FUNC(sceAppMgrConvertVs0UserDrivePath, source_path, dest_path, dest_len);
+    // stub. It looks working, but I'm not sure
```
Comment (Macdu) [inline vita3k/modules/SceDriverUser/SceAppMgrUser.cpp:145]: Can you add a STUBBED call?

File: vita3k/modules/SceNet/SceNet.cpp [hunk 2]
```diff
@@ -133,7 +201,9 @@ EXPORT(int, sceNetEpollWaitCB) {
 
 EXPORT(int, sceNetErrnoLoc) {
```
Comment (Macdu) [inline vita3k/modules/SceNet/SceNet.cpp:202]: Can you change the return value to a Ptr<int> ?

File: vita3k/modules/SceNetInternal/SceNetInternal.h [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+BRIDGE_DECL(sceNetSetDnsInfo)
+BRIDGE_DECL(sceNetSetsockopt)
+BRIDGE_DECL(sceNetShowIfconfig)
+BRIDGE_DECL(sceNetShowNetstat)
+BRIDGE_DECL(sceNetShowRoute)
+BRIDGE_DECL(sceNetShutdown)
+BRIDGE_DECL(sceNetSocket)
+BRIDGE_DECL(sceNetSocketAbort)
+BRIDGE_DECL(sceNetSocketClose)
+BRIDGE_DECL(sceNetTerm)
```
Comment (Macdu) [inline vita3k/modules/SceNetInternal/SceNetInternal.h:79]: What are all of these bridge declarations doing here? They are already in SceNet.h

File: vita3k/nids/include/nids/nids.inc [hunk 4]
```diff
@@ -5047,6 +5047,9 @@ NID(sceNetSocket, 0xF084FCE3)
 NID(sceNetSocketAbort, 0x891C1B9B)
 NID(sceNetSocketClose, 0x29822B4D)
 NID(sceNetTerm, 0xEA3CC286)
+// Library SceNetInternal
```
Comment (Macdu) [inline vita3k/nids/include/nids/nids.inc:5050]:
```suggestion
// Library "SceNetInternal"
```

**Comment by bookmist** [comment]
@Macdu I fixed bugs you found

**Comment by Macdu** [comment]
LGTM, the only "issue" is that now the only way to use the lle implementation of libHTTP and libSSL is to set the module list to manual, @EXtremeExploit are you ok with this?

**Review by Macdu — APPROVED** [review]

**Review by EXtremeExploit — APPROVED** [review]
sure

**Pull request merged**